instead of using OVERRIDEVERSIONCHECK, add a patch so fpc can be built
with the latest bugfix version as well

diff -wpruN '--exclude=*.orig' a~/Makefile.fpc a/Makefile.fpc
--- a~/Makefile.fpc	1970-01-01 00:00:00
+++ a/Makefile.fpc	1970-01-01 00:00:00
@@ -22,6 +22,7 @@ rule=help
 [prerules]
 REQUIREDVERSION=3.0.4
 REQUIREDVERSION2=3.2.0
+REQUIREDVERSION3=3.2.2
 
 
 # make versions < 3.77 (OS2 version) are buggy
@@ -261,6 +262,7 @@ versioncheckstartingcompiler:
 ifndef CROSSCOMPILE
 ifneq ($(FPC_VERSION),$(REQUIREDVERSION))
 ifneq ($(FPC_VERSION),$(REQUIREDVERSION2))
+ifneq ($(FPC_VERSION),$(REQUIREDVERSION3))
 ifndef OVERRIDEVERSIONCHECK
 	$(error The only supported starting compiler version is $(REQUIREDVERSION). You are trying to build with $(FPC_VERSION).)
 else
@@ -268,6 +270,7 @@ else
 endif
 endif
 endif
+endif
 endif
 
 ifeq ($(findstring $(CPU_TARGET),$(BuildOnlyBaseCPUs)),)
